docs/src/content/docs/pt/index.mdx

```diff
@@ -26,8 +26,8 @@ import MyIcon from '../../../components/MyIcon.astro';
 import SubscriptionForm from '../../../components/SubscriptionForm.astro';
 
 <CardGrid>
-  <Card title="54 Desafios">
```

Comment:
I think there is an issue with the generator, because the main index.mdx has not been updated.

Also you need to update line 16 with your new challenge for all index.mdx files. 
I will take a look at the generator to understand why it's not working properly.